tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AdoNet/SystemDataSqlClientTests.cs

```diff
@@ -25,16 +29,19 @@ public SystemDataSqlClientTests(ITestOutputHelper output)
         public static IEnumerable<object[]> GetEnabledConfig()
             => from packageVersionArray in PackageVersions.SystemDataSqlClient
                from metadataSchemaVersion in new[] { "v0", "v1" }
-               select new[] { packageVersionArray[0], metadataSchemaVersion };
+               from propagation in new[] { string.Empty, "100", "randomValue", "disabled", "service", "full" }
+               select new[] { packageVersionArray[0], metadataSchemaVersion, propagation };
 
         public override Result ValidateIntegrationSpan(MockSpan span, string metadataSchemaVersion) => span.IsSqlClient(metadataSchemaVersion);
 
         [SkippableTheory]
         [MemberData(nameof(GetEnabledConfig))]
         [Trait("Category", "EndToEnd")]
         [Trait("RunOnWindows", "True")]
-        public void SubmitsTraces(string packageVersion, string metadataSchemaVersion)
+        public async void SubmitsTraces(string packageVersion, string metadataSchemaVersion, string dbmPropagation)
```

Comment:
I think you can use `async Task` instead of `async void`. Using `async void` is generally not good practice, and the unit testing framework can probably use the returned `Task` to check for the test status (completed, error, etc).